modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -842,6 +844,8 @@ const Fields = ({
 						});
 
 						setFDSFields(newFDSFields);
+
+						updateFDSFieldsOrder();
```

Comment:
Same as above, I don't think this will actually work here. You'll need to verify that the request is including this new field id, but I don't think it will.